### ISSUE TITLE
'correct answer' button is not keyboard accessible

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2942]
Note: I'm specifically referring to navigating the course using the keyboard and NOT when using Jaws etc. Although it would be great if we could also look at getting that working, perhaps using an aria-live region to read out the correct answer(s)?

**Comment by moloko** [comment]
tested using v5.6.1 and v5.7.0 but almost certainly still an issue in v5.7.1

**Comment by oliverfoster** [comment]
As detailed previously and unless someone can think of an easier model: each question would need to be able to read its own correct answer as each question has a different format:

multi select mcq: you should have selected options 1, 3 and 5
single select mcq: you should have selected option 1
slider: you should have chosen between 1 and 4

etc

**Comment by moloko** [comment]
> At that point it's almost sensible to make this feedback visible as it is much more useful than what we have at the moment.

I think if we're talking about reading out the labels (G/MCQ) - which I think you'd need to do as 'option 1' then it would all start to get a bit crowded on-screen. and unnecessary since you could see the options you should have selected more quickly than reading them out. Matching questions would be even worse. So I think just having them read by the screen reader is a better option.

And yes totally understand it's something each question type will need to implement.

as a short-term solution, is it possible to make the 'correct answer' button keyboard accessible but still keep it hidden from assistive tech?

**Comment by oliverfoster** [comment]
> as a short-term solution, is it possible to make the 'correct answer' button keyboard accessible but still keep it hidden from assistive tech?

No, tab-index will allow focus regardless of the aria-hidden value. So it'd do nothing when focused / read anyway / mess up the screen reader.
